octopoes/bits/spf_discovery/spf_discovery.py

```diff
@@ -18,6 +19,10 @@
 def run(input_ooi: DNSTXTRecord, additional_oois: list, config: dict[str, Any]) -> Iterator[OOI]:
     if input_ooi.value.startswith("v=spf1"):
         spf_value = input_ooi.value.replace("%(d)", input_ooi.hostname.tokenized.name)
+
+        # remove exists:%i mechanisms
+        spf_value = re.sub(r"exists:%i[^\s]+", "", spf_value)
```

Comment:
The format is `%{i}`, but I think it would better to also remove all the other macros. This will remove also everything else up to the first dot after the macro

```suggestion
        spf_value = re.sub(r"exists:%{.*}[^.]*\.", "", spf_value)
```

Reply:
What do you mean with the first dot?

Reply:
This code removed the whole `exists:%{i}something.example.com` not only `%{i}something.example.com`

Reply:
There was a `\` missing, fixed it.